homeassistant/components/gree/const.py

```diff
@@ -0,0 +1,22 @@
+"""Constants for the Gree Climate integration."""
+
+DOMAIN = "gree"
+
+CONNECTION_NETWORK_MAC = "mac"
+
+DEFAULT_DISCOVERY = True
+DEFAULT_NAME = "Gree HVAC"
+DEFAULT_PORT = 7000
```

Comment:
Where are these used?

Reply:
Remnants of refactoring, they will be removed too.